Detectors/TPC/simulation/src/Detector.cxx

```diff
@@ -326,6 +328,23 @@ void Detector::CreateMaterials()
 
   Float_t density;
 
+  // TODO: load pressure and temperature values from CCDB
+  const double pressure = gasParam.Pressure;                // in mbar
+  const double temperature = gasParam.Temperature + 273.15; // in K
+
+  // densities were taken for these values
+  const Double_t t1 = 293.15;  // 20°C in K
+  const Double_t p1 = 1013.25; // 1 atm in mbars
```

Comment:
use either consistently `double`, or `Double_t`. Since most places use the latter, I suggest to stick with it.